### PR TITLE
Update expression.dd

### PR DESCRIPTION
**Pull request by RazvanN7**
Part 2 of #1917

### CODE REVIEW

File: spec/expression.dd [hunk 1]
```diff
@@ -110,13 +110,18 @@ $(GRAMMAR
...
+    $(GNAME IfTrue):
+       $(EXPRESSION)
 )
 
     $(P The first expression is converted to $(D bool), and is evaluated.
     )
 
-    $(P If it is $(D true), then the second expression is evaluated, and
-        its result is the result of the conditional expression.
+    $(P If it is $(D true) and  the second expression is provided, then
```
Comment (jacob-carlborg) [inline spec/expression.dd]: There are two spaces between `and` and `the`.

Comment (andralex) [reply spec/expression.dd]: @jacob-carlborg don't forget you can edit PRs directly for such things (I'll do it just now)

Comment (jacob-carlborg) [reply spec/expression.dd]: I haven't tested this but the documentation says:

"You can only make commits on pull request branches that: are opened in a repository that you have push access to and that were created from a fork of that repository" [1]

[1] https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by andralex** [comment]
Closing pending DIP if anyone takes it.

**Pull request closed without merging**